### DISCUSSION TITLE
HwV380E12_WF3_PCARD_LIGHT_20160217

### DISCUSSION THREAD

**Discussion by pingflood** [Arkady23/V380-Web#44] [General]
Hello my friend,

I'm here to report another camera data.

I have tested this V380 Light Bulb model: https://www.aliexpress.com/item/1005004046916223.html

V380 App identify it as:
AppV380E12_PCARD_LIGHT_V2.5.10.6_20180308
KerV380E12_PCARD_LIGHT_V2.1.0_20160506
HwV380E12_WF3_PCARD_LIGHT_20160217

Wireshark data:
{"soft_id":"V380E2_CA","dev_id":,"version":2051006,"timestamp":1654345953 }

And it works perfectly with current 
V380 | HwV380E12_WF3_PTZ / HwV380E13_WF3_PTZ | V380E2_CA | httpd_V380E2_CA.zip

Hope this info helps your amazing project.

Kind regards

**Comment by Arkady23** [comment]
Added a description for installing a web server manually for new cameras.
